.code-samples.meilisearch.yaml

```
@@ -8,7 +8,7 @@ get_one_index_1: |-
 list_all_indexes_1: |-
   $client->getAllIndexes();
 create_an_index_1: |-
-  $client->createIndex('movies', ['primaryKey' => 'movie_id']);
+  $client->createIndex('movies', ['primaryKey' => 'id']);
 update_an_index_1: |-
   $client->updateIndex('movies', ['primaryKey' => 'movie_review_id']);
```

```suggestion
  $client->updateIndex('movies', ['primaryKey' => 'id']);
```

should also be removed two lined beneath